Source/Core/Common/MemoryUtil.cpp

```diff
@@ -88,20 +88,21 @@ void FreeMemoryPages(void* ptr, size_t size)
 {
   if (ptr)
   {
-    bool error_occurred = false;
-
 #ifdef _WIN32
     if (!VirtualFree(ptr, 0, MEM_RELEASE))
-      error_occurred = true;
+    {
+      PanicAlert("FreeMemoryPages failed!\n"
+                 "VirtualFree: %s",
+                 GetLastErrorMsg().c_str());
```

Comment:
If you're talking about the parameters, no, according to lint. That's why I added the braces.

If you're talking about these strings I can put them into a single one even if I don't really find them easy to read with the "\n" in the middle